### DISCUSSION TITLE
SetOutParam:AbsXg is not an available output channel.

### DISCUSSION THREAD

**Discussion by aitorplaza** [OpenFAST/openfast#1401] [General]
Dear everyone,

I'm trying to output the absolute node position whit BeamDyn. I'm using the last version, fresh compiled by me on Ubuntu 22.04. 

I'm just using the code from the reg-test MW_Land_BD_DLL_WTurb and I've added the following output channels to NRELOffshrBsline5MW_BeamDyn.dat file: "AbsXg", "AbsYg", "AbsZg", "AbsXr", "AbsYr", "AbsZr" as shown in the picture:

![image](https://i.imgur.com/NUKXywi.png)

When running OpenFASt I'm getting the following error: 

`FAST_InitializeAll:BD_Init:SetParameters:SetOutParam:AbsXg is not an available output channel.`

Prior to adding these entries the code works perfectly. Probably I'm doing something wrong, can you help me? 

Thanks in advance

**Comment by jjonkman** [comment]
Dear @aitorplaza,

BeamDyn outputs "`AbsXg`" etc. are not available in the normal write output list.  Rather, these outputs are available in the "Nodal Outputs" section of the BeamDyn primary input file, which should appear after the "`END`" line of the input file.  See the "Nodal Outputs" section of the BeamDyn documentation here for more information: https://openfast.readthedocs.io/en/main/source/user/beamdyn/input_files.html#nodal-outputs.

Best regards,

**Comment by aitorplaza** [comment]
Thank you very much @jjonkman , as expected I was doing something wrong.
